src/Language/Mimsa/Server/Handlers.hs

```diff
@@ -0,0 +1,157 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
Most of the endpoints are doing the same old shit over and over, this pulls out the key functions into `Handler` types.